src/reynir_correct/errtokenizer.py

```diff
@@ -928,13 +956,17 @@ def get() -> CorrectToken:
                     yield token
                 else:
                     # Step to next token
-                    next_token = CorrectToken.word(token.txt)
+                    compspan = token.origin_spans
+                    for _ in range(next_token.origin_spans[0]):
```

Comment:
Better: `compspan.extend([0] * next_token.origin_spans[0])` (instead of for loop)